Dockerfile

```diff
@@ -61,7 +61,7 @@ WORKDIR $GOPATH/src/github.com/go-swagger/go-swagger
 # Install go-swagger for validating swagger.yaml
 # This is https://github.com/kolyshkin/go-swagger/tree/golang-1.13-fix
 # TODO: move to under moby/ or fix upstream go-swagger to work for us.
```

Comment:
It's hurting my head to read (because I do not Swagger nearly enough), but do you think it's worth adding a link to https://github.com/go-swagger/go-swagger/issues/2077 in the comments here?

(there are some suggestions in there that seem like they'd be good fodder for a "help wanted" if someone wanted to pitch in here, so it seems like a shame _not_ to link to :sweat_smile:)

Reply:
I created https://github.com/moby/moby/issues/42593 earlier today, and want to look if I can get someone to work on that in our next sprint